component/initSoftRobots.cpp

```diff
@@ -95,7 +95,7 @@ const char* getModuleComponentList()
 }
 
 SOFA_LINK_CLASS(AnimationEditor)
-SOFA_LINK_CLASS(CommunicationController)
+//SOFA_LINK_CLASS(CommunicationController)
```

Comment:
Commenting dead feature is a bad practice. Don't commit it. 
A small explanation why:
https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f